### PR TITLE
fix(ha): skip interfaces configuration if mode is MANUAL

### PR DESCRIPTION
**Pull request by william-gr**
Ticket:	#27202

### CODE REVIEW

File: src/freenas/etc/rc.conf.local [hunk 1]
```diff
@@ -158,6 +158,9 @@ _interface_config()
 	# configuration.
 	${FREENAS_SQLITE_CMD} ${FREENAS_CONFIG} "SELECT id, int_interface, int_dhcp, int_ipv4address, int_ipv4address_b, int_v4netmaskbit, int_ipv6auto, int_ipv6address, int_v6netmaskbit, int_vip, int_vhid, int_pass, int_critical, int_group, int_options FROM network_interfaces ORDER BY id" | \
 	while read interface_id interface dodhcp ipv4addr ipv4addr_b ipv4netmask doipv6auto ipv6addr ipv6netmask int_vip int_vhid int_pass critical group options; do
+		if [ "$(ha_mode)" = "MANUAL" ]; then
+			continue;
```
Comment (agokhale) [inline src/freenas/etc/rc.conf.local]: please syslog that interface configuration was aborted due to manual ha mode/ chassis detection failure ... debugging why the interface did not configure will be difficult unless we have an indicatior of why we got here.

Comment (william-gr) [reply src/freenas/etc/rc.conf.local]: @agokhale how do you propose we do that? This code runs early enough in the boot syslogd has not started.

**Comment by amotin** [comment]
Won't this be too aggressive, completely blocking network in case somebody try to run TrueNAS on unknown hardware or, for example, inside VM for testing?  May be we should differentiate that based on presence of HA license or something?  Sorry if I miss something, but I don't see that checked now.

**Comment by william-gr** [comment]
@amotin you're right. I have not thought about this, how about now? @agokhale

**Comment by agokhale** [comment]
Logging to console at boot time is sufficient. If we need to check license entitlements  before attempting/failing at hardware detection to allow developement,  that's fine too.

**Comment by amotin** [comment]
> @amotin you're right. I have not thought about this, how about now?

This looks fine to me.  Just code lines of 150+ chars in length make me sad.

**Pull request merged**